### PR TITLE
Installer for Windows and Open Prompt

### PR DESCRIPTION
**Pull request by My12123**
All files are checked, work correctly.
Video test
https://terabox.com/s/1y1JaeyHPc2IvGoV-tYBobQ
https://wdho.ru/6Rtz

### CODE REVIEW

**Comment by My12123** [comment]
All files are checked, work correctly.
Video test
https://terabox.com/s/1y1JaeyHPc2IvGoV-tYBobQ
https://wdho.ru/6Rtz

**Comment by My12123** [comment]
@Kiteretsu77 When will it be approved?

**Comment by Kiteretsu77** [comment]
> @Kiteretsu77 When will it be approved?

I just check the video link, I actually cannot open the video link.

File: Installer for Windows. Open for reading in the editor.bat
[the remark this answers is not in the document]
Comment (My12123) [reply Installer for Windows. Open for reading in the editor.bat:1]: @Kiteretsu77 
Everything works AMD and Intel.

I have Intel, everything was installed in the test.

**Pull request merged**